core/dbt/parser/schema_test_builders.py

```diff
@@ -224,13 +224,20 @@ def __init__(
         groups = match.groupdict()
         self.name: str = groups['test_name']
         self.namespace: str = groups['test_namespace']
-        self.modifiers: Dict[str, Any] = {}
-        for key in self.MODIFIER_ARGS:
+        self.config: Dict[str, Any] = {}
```

Comment:
Need to add support for `config:` dict for `unique` + `not_null` as well. This currently raises an error:
```yml
- unique:
    config:
      severity: warn
```
```
macro 'dbt_macro__test_unique' takes no keyword argument 'config'
```

Reply:
I can't recreate this. You're sure you were using the branch?

Reply:
I must've been doing something funny! Just tried this again with the latest version of this branch, and it works just fine